### PR TITLE
Fix issue #1426 where remote poller was not using unique filenames

### PR DESCRIPTION
**Pull request by netniV**
Fix for issue #1426 where remote poller was not using unique filenames when attempting to verify files

### CODE REVIEW

File: lib/poller.php [hunk 1]
```diff
@@ -879,7 +874,12 @@ function resource_cache_out($type, $path) {
 											cacti_log("ERROR: Cache in cannot write to '" . $mypath . "', purge this location");
 										}
 									} else {
-										cacti_log("ERROR: PHP Source File '" . $mypath . "' from Cache has a Syntax error!", false, 'POLLER');
+										cacti_log("ERROR: PHP Source File '" . $mypath . "' from Cache has an error whilst checking syntax ($exit)!", false, 'POLLER');
+										cacti_log("ERROR: PHP Source File '" . $mypath . "' tried to execute: $php_path -l $tmpfile", false, 'POLLER');
+										$lines = explode("\n",$output);
+										foreach ($lines as $line) {
+											cacti_log("ERROR: PHP Source File '" . $mypath . "': $line!", false, 'POLLER');
+										}
```
Comment (cigamit) [inline lib/poller.php]: I would prefer one line output here for a single event.  Can you make that change?

Comment (netniV) [reply lib/poller.php]: Use the clean_up_lines() function instead of outputing multiple lines? Or just leave the write as it was?

Comment (cigamit) [reply lib/poller.php]: Remove line breaks and carriage returns.  Make it just one line.

**Comment by netniV** [comment]
I have now condensed the output rather than issues multiple lines.

File: lib/poller.php [hunk 2]
```diff
@@ -879,7 +874,9 @@ function resource_cache_out($type, $path) {
 											cacti_log("ERROR: Cache in cannot write to '" . $mypath . "', purge this location");
 										}
 									} else {
-										cacti_log("ERROR: PHP Source File '" . $mypath . "' from Cache has a Syntax error!", false, 'POLLER');
+										cacti_log("ERROR: PHP Source File '" . $mypath . "' from Cache has an error whilst checking syntax ($exit)!", false, 'POLLER');
+										cacti_log("ERROR: PHP Source File '" . $mypath . "' tried to execute: $php_path -l $tmpfile", false, 'POLLER');
+										cacti_log("ERROR: PHP Source File '" . $mypath . "': " . str_repalce("\n"," ",str_replace("\n"," ", $output)), false, 'POLLER');
```
Comment (cigamit) [inline lib/poller.php]: Oops str_repalce?

Comment (netniV) [reply lib/poller.php]: Ack.

Comment (netniV) [reply lib/poller.php]: That's what I get for rushing it around tea time.

**Comment by netniV** [comment]
I have now updated the code.  I believe we have a choice though, leave it as two lines to avoid confusion between the execution and output, or remove either the execution or output.

**Pull request merged**